app/utils.py

```diff
@@ -29,3 +30,54 @@ def parse_phone_number(phone_number):
         stripped = '1' + stripped
     stripped = '+' + stripped
     return stripped
+
+
+def parse_to_db(db, filename):
+    city_default = ', philadelphia, pennsylvania, usa'
```

Comment:
I would check if `geocoder` supports this and if it doesn't we may have to use the geocoding api directly :(